### PR TITLE
Bp 2620 Rename creditcards into cards

### PR DESCRIPTION
**Pull request by harli91**

### CODE REVIEW

**Review by vegimcarkaxhija — APPROVED** [review]

File: gateway-buckaroo-creditcard.php [hunk 1]
```diff
@@ -386,16 +391,16 @@ public function init_form_fields()
         parent::init_form_fields();
 
         $this->form_fields['creditcardmethod'] = array(
-            'title'       => __('Credit card method', 'wc-buckaroo-bpe-gateway'),
+            'title'       => __('Card method', 'wc-buckaroo-bpe-gateway'),
```
Comment (Buckaroo-Rene) [inline gateway-buckaroo-creditcard.php:394]: 'title'       => __('Credit card and debit card method', 'wc-buckaroo-bpe-gateway'),

File: gateway-buckaroo-creditcard.php [hunk 2]
```diff
@@ -386,16 +391,16 @@ public function init_form_fields()
...
             'type'        => 'select',
-            'description' => __('Redirect user to Buckaroo or enter creditcard information inline in the checkout. SSL is required to enable inline creditcard information.', 'wc-buckaroo-bpe-gateway'),
+            'description' => __('Redirect user to Buckaroo or enter card information inline in the checkout. SSL is required to enable inline creditcard information.', 'wc-buckaroo-bpe-gateway'),
```
Comment (Buckaroo-Rene) [inline gateway-buckaroo-creditcard.php:396]: 'description' => __('Redirect the user to Buckaroo or enter card information directly (inline) in the checkout. SSL is required to enable inline creditcard information.', 'wc-buckaroo-bpe-gateway'),

File: gateway-buckaroo-creditcard.php [hunk 3]
```diff
@@ -386,16 +391,16 @@ public function init_form_fields()
...
             'options'     => array('redirect' => 'Redirect', 'encrypt' => 'Inline'),
             'default'     => 'encrypt',
             'desc_tip'    =>__('Check with Buckaroo whether Client Side Encryption is enabled, otherwise transactions will fail. If in doubt, please contact us.', 'wc-buckaroo-bpe-gateway'),
         
         );
         $this->form_fields['creditcardpayauthorize'] = array(
-            'title'       => __('Credit card Pay or Capture', 'wc-buckaroo-bpe-gateway'),
+            'title'       => __('Pay or Capture', 'wc-buckaroo-bpe-gateway'),
```
Comment (Buckaroo-Rene) [inline gateway-buckaroo-creditcard.php:403]: 'title'       => __('Credit and debit card flow', 'wc-buckaroo-bpe-gateway'),

File: gateway-buckaroo-creditcard.php [hunk 4]
```diff
@@ -416,7 +421,7 @@ public function init_form_fields()
                     'visaelectron'   => 'Visa Electron',
                     'vpay'           => 'Vpay'
                 ),
-                'description' => __('Select which Creditcard providers  will be visible to customer', 'wc-buckaroo-bpe-gateway'),
+                'description' => __('Select which card providers will be visible to customer', 'wc-buckaroo-bpe-gateway'),
```
Comment (Buckaroo-Rene) [inline gateway-buckaroo-creditcard.php:424]: 'description' => __('Select which credit or debit card providers will be visible to customer', 'wc-buckaroo-bpe-gateway'),

File: gateway-buckaroo-creditcard.php [hunk 5]
```diff
@@ -448,7 +453,7 @@ public function init_form_fields()
                     'visaelectron'   => 'Visa Electron',
                     'vpay'           => 'Vpay'
                 ),
-                'description' => __('Select which Creditecard providers will be shown separate in checkout', 'wc-buckaroo-bpe-gateway'),
+                'description' => __('Select which card provider will be shown separate in checkout', 'wc-buckaroo-bpe-gateway'),
```
Comment (Buckaroo-Rene) [inline gateway-buckaroo-creditcard.php:456]: 'description' => __('Select which credit and debit card providers will be shown separately in the checkout', 'wc-buckaroo-bpe-gateway'),

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 6]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
+msgid "Enable %s Payment Method"
+msgstr ""
+
+#: gateway-buckaroo-afterpay.php:429
 msgid "Enable B2B option for Riverty | AfterPay"
 msgstr ""
 
-#: gateway-buckaroo.php:346
+#: gateway-buckaroo-paypal.php:117
+msgid "Enable paypal express for the following pages."
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:336]: msgid "Enable PayPal express for the following pages."

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 7]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
+
+#: library/common.php:280 library/common.php:303
 msgid ""
 "Payment unsuccessful. Please try again or choose another payment method."
 msgstr ""
 
-#: gateway-buckaroo.php:440
-msgid "Percentage or static"
+#: gateway-buckaroo-paypal.php:115
+msgid "Paypal express"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:613]: msgid "PayPal express"

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 8]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
 msgstr ""
 
-#: gateway-buckaroo-afterpay.php:1008 gateway-buckaroo-afterpaynew.php:1075
-#: gateway-buckaroo-klarna.php:478
+#: gateway-buckaroo-paypal.php:110
+msgid "Paypal express merchant id"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:617]: msgid "PayPal express merchant id"

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 9]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
+msgstr ""
+
+#: controllers/PaypalExpressController.php:101
+msgid "Paypal merchant id is required"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:621]: msgid "PayPal merchant id is required"

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 10]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
+msgstr ""
+
+#: gateway-buckaroo-paypal.php:112
+msgid "Paypal merchant id required for paypal express"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:625]: msgid "PayPal merchant id required for paypal express"

File: languages/wc-buckaroo-bpe-gateway-de_DE.po [hunk 11]
```diff
@@ -14,1004 +14,1085 @@ msgstr ""
...
+#: gateway-buckaroo-paypal.php:120
+msgid "Product page"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:628
+#: gateway-buckaroo-creditcard.php:396
 msgid ""
-"Redirect user to Buckaroo or enter creditcard information inline in the "
-"checkout. SSL is required to enable inline creditcard information"
+"Redirect user to Buckaroo or enter card information inline in the checkout. "
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-de_DE.po:704]: "Redirect user to Buckaroo or enter credit or debit card information inline in the checkout. "

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 12]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
 msgstr ""
 
-#: gateway-buckaroo-afterpay.php:1008 gateway-buckaroo-afterpaynew.php:1075
-#: gateway-buckaroo-klarna.php:478
+#: controllers/PaypalExpressController.php:101
+msgid "Paypal merchant id is required"
+msgstr ""
+
+#: gateway-buckaroo-paypal.php:112
+msgid "Paypal merchant id required for paypal express"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po]: msgid "PayPal merchant id required for paypal express"

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 13]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
+msgstr ""
+
+#: gateway-buckaroo-paypal.php:110
+msgid "Paypal express merchant id"
 msgstr ""
 
-#: gateway-buckaroo-afterpay.php:1008 gateway-buckaroo-afterpaynew.php:1075
-#: gateway-buckaroo-klarna.php:478
+#: controllers/PaypalExpressController.php:101
+msgid "Paypal merchant id is required"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po]: msgid "PayPal merchant id is required"

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 14]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
 msgstr ""
 
-#: gateway-buckaroo.php:440
-msgid "Percentage or static"
+#: gateway-buckaroo-paypal.php:115
+msgid "Paypal express"
+msgstr ""
+
+#: gateway-buckaroo-paypal.php:110
+msgid "Paypal express merchant id"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po]: msgid "PayPal express merchant id"

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 15]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
+
+#: library/common.php:280 library/common.php:303
 msgid ""
 "Payment unsuccessful. Please try again or choose another payment method."
 msgstr ""
 
-#: gateway-buckaroo.php:440
-msgid "Percentage or static"
+#: gateway-buckaroo-paypal.php:115
+msgid "Paypal express"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po]: msgid "PayPal express"

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 16]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
+msgstr ""
+
+#: gateway-buckaroo-mastersettings.php:59
 msgid "Secret key"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:239
+#: gateway-buckaroo-creditcard.php:226
 msgid "Select a credit card."
 msgstr "Sélectionnez une carte de crédit"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po:743]: msgstr "Sélectionnez une carte de crédit ou de débit"

File: languages/wc-buckaroo-bpe-gateway-fr_FR.po [hunk 17]
```diff
@@ -14,769 +14,876 @@ msgstr ""
...
+msgstr ""
+
+#: gateway-buckaroo-creditcard.php:456
+msgid "Select which card provider will be shown separate in checkout"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:667
-msgid "Select which Creditcard providers  will be visible to customer"
+#: gateway-buckaroo-creditcard.php:424
+msgid "Select which card providers will be visible to customer"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-fr_FR.po]: msgid "Select which credit or debit card providers will be visible to customer"

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 18]
```diff
@@ -154,22 +158,35 @@ msgstr ""
...
-#: library/common.php:114
+#: library/common.php:147
 #, fuzzy, php-format
 #| msgid "Captured %s - Capture transaction ID: %s"
 msgid "Captured %1$s - Capture transaction ID: %2$s"
 msgstr "Captured %s - Capture transactie-ID: %s"
 
-#: library/common.php:413
+#: gateway-buckaroo-creditcard.php:394
+msgid "Card method"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po]: msgid "Credit and debit card method"

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 19]
```diff
@@ -154,22 +158,35 @@ msgstr ""
...
+msgstr "Creditcard"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po:187]: msgstr "Credit en debetkaart"

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 20]
```diff
@@ -734,51 +729,62 @@ msgstr "Richt je camera op de QR-code."
...
-#: gateway-buckaroo-creditcard.php:391
+#: gateway-buckaroo-creditcard.php:396
+#, fuzzy
+#| msgid ""
+#| "Redirect user to Buckaroo or enter card information inline in the "
+#| "checkout. SSL is required to enable inline creditcard information"
 msgid ""
-"Redirect user to Buckaroo or enter creditcard information inline in the "
-"checkout. SSL is required to enable inline creditcard information"
+"Redirect user to Buckaroo or enter card information inline in the checkout. "
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po]: "Redirect user to Buckaroo or enter credit or debit card information (directly) inline in the checkout. "

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 21]
```diff
@@ -734,51 +729,62 @@ msgstr "Richt je camera op de QR-code."
...
+"SSL is required to enable inline creditcard information."
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po:739]: "SSL is required to enable inline credit or debit card information."

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 22]
```diff
@@ -803,23 +809,25 @@ msgstr "Selecteer belastingklasse voor de betaaltoeslag"
...
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:419
-msgid "Select which Creditcard providers  will be visible to customer"
-msgstr "selecteer welke Creditecard-aanbieders aan de klant verschijnen"
-
-#: gateway-buckaroo-creditcard.php:451
-msgid "Select which Creditecard providers will be shown separate in checkout"
+#: gateway-buckaroo-creditcard.php:456
+msgid "Select which card provider will be shown separate in checkout"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po:813]: msgid "Select which credit or debit card providers will be shown separately in the checkout"

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 23]
```diff
@@ -803,23 +809,25 @@ msgstr "Selecteer belastingklasse voor de betaaltoeslag"
...
 msgstr ""
 
-#: gateway-buckaroo-payperemail.php:192
+#: gateway-buckaroo-creditcard.php:424
+#, fuzzy
+#| msgid "Select which Creditcard providers  will be visible to customer"
+msgid "Select which card providers will be visible to customer"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po]: msgid "Select which credit and debit card providers will be visible to customer"

File: languages/wc-buckaroo-bpe-gateway-nl_NL.po [hunk 24]
```diff
@@ -803,23 +809,25 @@ msgstr "Selecteer belastingklasse voor de betaaltoeslag"
...
+msgstr "selecteer welke Creditecard-aanbieders aan de klant verschijnen"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway-nl_NL.po:820]: msgstr "Selecteer welke credit of debetkaart-aanbieders voor de klant verschijnen"

File: languages/wc-buckaroo-bpe-gateway.pot [hunk 25]
```diff
@@ -718,48 +703,55 @@ msgstr ""
 msgid "Product page"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:391
+#: gateway-buckaroo-creditcard.php:396
 msgid ""
-"Redirect user to Buckaroo or enter creditcard information inline in the "
-"checkout. SSL is required to enable inline creditcard information"
+"Redirect user to Buckaroo or enter card information inline in the checkout. "
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway.pot]: "Redirect user to Buckaroo or enter credit or debit card information directly (inline) in the checkout. "

File: languages/wc-buckaroo-bpe-gateway.pot [hunk 26]
```diff
@@ -718,48 +703,55 @@ msgstr ""
...
+"SSL is required to enable inline creditcard information."
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway.pot:709]: "SSL is required to enable inline credit and debit card information."

File: languages/wc-buckaroo-bpe-gateway.pot [hunk 27]
```diff
@@ -718,48 +703,55 @@ msgstr ""
...
+msgid "Riverty | AfterPay Pay or Capture"
+msgstr ""
+
 #: gateway-buckaroo-mastersettings.php:59
 msgid "Secret key"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:221
+#: gateway-buckaroo-creditcard.php:226
 msgid "Select a credit card."
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway.pot:747]: msgid "Select a credit or debit card."

File: languages/wc-buckaroo-bpe-gateway.pot [hunk 28]
```diff
@@ -781,23 +773,23 @@ msgstr ""
 msgid "Select where to store debug data"
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:419
-msgid "Select which Creditcard providers  will be visible to customer"
+#: gateway-buckaroo-creditcard.php:456
+msgid "Select which card provider will be shown separate in checkout"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway.pot:777]: msgid "Select which credit and debit card provider will be shown separately in the checkout"

File: languages/wc-buckaroo-bpe-gateway.pot [hunk 29]
```diff
@@ -781,23 +773,23 @@ msgstr ""
...
 msgstr ""
 
-#: gateway-buckaroo-creditcard.php:451
-msgid "Select which Creditecard providers will be shown separate in checkout"
+#: gateway-buckaroo-creditcard.php:424
+msgid "Select which card providers will be visible to customer"
```
Comment (Buckaroo-Rene) [inline languages/wc-buckaroo-bpe-gateway.pot]: msgid "Select which credit and debit card providers will be visible for the customer"

**Review by Buckaroo-Rene — APPROVED** [review]
Translations looks good.

**Review by vegimcarkaxhija — APPROVED** [review]

**Pull request merged**